packages/neuron-ui/src/containers/Footer/index.tsx

```diff
@@ -31,6 +31,10 @@ export const SyncStatus = ({
     return <Text variant="xSmall">{t('footer.fail-to-fetch-tip-block-number')}</Text>
   }
 
+  if (+syncedBlockNumber < 0) {
```

Comment:
Please using bigint for `blockNumber: u64`

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/1126/commits/cc3cd039c832bdccbe5b7d9a770fd78b5255f34f